keps/sig-release/NNNN-reducing-build-maintenance/kep.yaml

```diff
@@ -0,0 +1,15 @@
+title: Reducing Build Maintenance in CIP
+kep-number: NNNN
```

Comment:
let's update this here with  "2818" along with the file names as well :)

Reply:
Fixed!